### PR TITLE
Replaced all references to "master"

### PR DESCRIPTION
**Pull request by punkdata**
Submitting this to PR to replace the term master on this page to Primary. I would suggest the community replace all the `master` references moving forward. #BlackLivesMatter

### CODE REVIEW

**Comment by punkdata** [comment]
"I signed it"

**Comment by sftim** [comment]
Thanks for this PR @punkdata 

Relevant to issue #21749 
/hold

pending discussion on that issue and elsewhere

**Comment by sftim** [comment]
Ah, maybe not _all_ references to “master”.
/hold cancel

File: content/en/docs/concepts/_index.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ To work with Kubernetes, you use *Kubernetes API objects* to describe your clust
...
 
-* The **Kubernetes Master** is a collection of three processes that run on a single node in your cluster, which is designated as the master node. Those processes are: [kube-apiserver](/docs/admin/kube-apiserver/), [kube-controller-manager](/docs/admin/kube-controller-manager/) and [kube-scheduler](/docs/admin/kube-scheduler/).
-* Each individual non-master node in your cluster runs two processes:
-  * **[kubelet](/docs/admin/kubelet/)**, which communicates with the Kubernetes Master.
+* The **Kubernetes Primary** is a collection of three processes that run on a single node in your cluster, which is designated as the primary node. Those processes are: [kube-apiserver](/docs/admin/kube-apiserver/), [kube-controller-manager](/docs/admin/kube-controller-manager/) and [kube-scheduler](/docs/admin/kube-scheduler/).
+* Each individual non-primary node in your cluster runs two processes:
```
Comment (sftim) [inline content/en/docs/concepts/_index.md:23]: This needs a bit more rewording (see issue #21800): these processes don't actually have to run on a single host.

Comment (stefanlasiewski) [reply content/en/docs/concepts/_index.md:23]: Is _primary_ the preferred term to replace _master_ elsewhere in the docs?

Comment (sftim) [reply content/en/docs/concepts/_index.md:23]: > Is _primary_ the preferred term to replace _master_ elsewhere in the docs?

“It depends”. _primary_ pairs well with _replica_, but sometimes it is better to replace _master_ with _control plane_, or other wording (“leader” if it's the result of an election, for example).

Comment (tengqm) [reply content/en/docs/concepts/_index.md:23]: Well ... IMHO, "Kubernetes Control Plane" is the term we use elsewhere and the preferred replacement.

Comment (stefanlasiewski) [reply content/en/docs/concepts/_index.md:23]: In other places throughout the documentation, the **Master** (I'll refer to this as the **Primary** down below) is discussed as *one piece* of the Kubernetes Control Plane, not a direct 1-to-1 replacement. The Kubelet is also part of the Control Plane but is not part of the Master.

If we were simply to replace the word "master" with the word "Kubernetes Control Plane", then we end up with awkward descriptions such as this at  https://kubernetes.io/docs/concepts/#kubernetes-control-plane

> The various parts of the Kubernetes Control Plane, such as the Kubernetes Master and kubelet processes

becomes the following, which doesn't make any sense:

> The various parts of the Kubernetes Control Plane, such as the Kubernetes Control Plane and kubelet processes

We can of course re-word this to merge the Primary & CP concepts, but we need to be careful not to erase this architectural distinction. The Kubernetes documentation already has a lot of inconsistencies which are frustrating & confusing for new users. See #21755 for a similar example.

In that case, replacing "Master" with "Primary" would be the easy fix, as long the term "Primary" is the agreed-upon term and is used consistently throughout the documentation.

Comment (sftim) [reply content/en/docs/concepts/_index.md:23]: BTW, I don't count the kubelet as part of the control plane - see for example the diagram at https://kubernetes.io/docs/concepts/overview/components/#editPageButton

Comment (sftim) [reply content/en/docs/concepts/_index.md:23]: I think I'm going to rewrite https://kubernetes.io/docs/concepts/#kubernetes-control-plane anyway.

**Comment by zparnold** [comment]
@punkdata Could you rebase your changes on the main branch for this repo (currently `master`) we'd love to get this change going!

**Comment by punkdata** [comment]
> @punkdata Could you rebase your changes on the main branch for this repo (currently `master`) we'd love to get this change going!

**Comment by punkdata** [comment]
> @punkdata Could you rebase your changes on the main branch for this repo (currently `master`) we'd love to get this change going!

@zparnold MCs fixed and pushed thx

**Pull request closed without merging**